backend/src/main/resources/db/changelog/db.changelog.2023-01-22-home-away-as-enum-type.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+      <changeSet author="juliusvandis (generated)" id="1674399959000-1">
+        <sql>CREATE TYPE home_away AS ENUM ('HOME','AWAY')</sql>
+    </changeSet>
+
+    <changeSet author="juliusvandis (generated)" id="1674399959000-2">
+        <modifyDataType  columnName="home_away"
+                         newDataType="home_away"
+                         schemaName="public"
+                         tableName="match"/>
+    </changeSet>
```

Comment:
Go through codebase and apply same logic (introduce enum types in db) over there. `Position` for User is a good candidate. There might be more.